linera-sdk/src/test/validator.rs

```diff
@@ -201,10 +201,19 @@ impl TestValidator {
 
         let admin_chain = self.get_chain(&ChainId::root(0));
 
+        let committee_blob = Blob::new(BlobContent::new_committee(
+            bcs::to_bytes(&committee).unwrap(),
+        ));
+        let blob_hash = committee_blob.id().hash;
+        self.storage
+            .write_blob(&committee_blob)
```

Comment:
I think changing the resource policy should not require re-publishing a new committee. There should be a separate mechanism for that. If we change the cost of a Wasm multiplication will we also publish a new committee? It should be stored somewhere in the "system storage" and its updates should require quorum of validators.

Reply:
But it also needs to be applied separately to each chain, just like committee changes. It makes sense to me to treat all changes with that property as one thing. Anyway, this wasn't changed in this PR, so it's probably a separate discussion.